samples/brokers/coap-broker/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "coap-broker"
+version = "0.1.0"
```

Comment:
should the versions match our overall versioning ("0.6.14" ... or better yet "0.7.0")?

Reply:
and update version.sh for each new Cargo.toml verion location?